zanata-war/src/main/webapp/WEB-INF/pages.xml

```diff
@@ -172,62 +43,16 @@
     </navigation>
   </page>
 
-
   <!-- Admin -->
   <!-- TODO [CDI] find a way to secure a url pattern in CDI -->
   <page view-id="/admin/*" action="#{identity.checkRole('admin')}"></page>
```

Comment:
Worst case scenario we have to list out every single admin view in faces-config.xml